core/spec/models/spree/shipment_spec.rb

```diff
@@ -39,7 +39,7 @@
           end
         end.to change { Spree::Shipment.count }.by(1)
 
-        new_shipment = order.shipments.last
+        new_shipment = (order.shipments.to_a - [shipment])[0]
```

Comment:
Why not `order.shipments.order(:created_at).last` or `order.shipments.sort_by(&:created_at).last` if we don't want to reload the shipments again? Would be more readable.

Reply:
I did this out of habit due to MySQL second timestamp issues, but this isn't necessary as of #2598 :tada:. I will use your suggestion.

Reply:
Ah. Thanks for clarifying. I always forget the darn mysql default precision.